### PR TITLE
Made responsive for ipad and ipad pro

### PR DESCRIPTION
**Pull request by sheetalj2205**
sam
ipad pro
ipad

## Related Issue

- Info about the related issue

Closes: #328 

### Describe the changes you've made
made responsive
Give a clear description what modifications you have made

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Describe how have you verified the changes made

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |

### CODE REVIEW

**Comment by msk4862** [comment]
@sheetalj2205 Please add screenshots too.

File: src/components/Carousel.jsx [hunk 1]
```diff
@@ -13,21 +13,21 @@ export default function Carousel({ data }) {
...
-    setIsAnimating(true);
-    setTimeout(() => {
-      let newIndex;
-      if (activeIndex === 0) {
-        newIndex = data.length - 1;
-      } else newIndex = activeIndex - 1;
-      setActiveIndex(newIndex);
-      setActiveQuote(data[newIndex]);
-    }, ANIMATION_DURATION / 2);
+  // function goToPrevious() {
```
Comment (Anurag5086) [inline src/components/Carousel.jsx:16]: remove comments

**Review by Anurag5086 — CHANGES_REQUESTED** [review]

File: src/components/Carousel.jsx (hunk 1 above)
Comment (sheetalj2205) [reply src/components/Carousel.jsx:16]: I haven't made any changes in carousel.jsx.

**Review by msk4862 — COMMENTED** [review]
@sheetalj2205 add one screenshot for non-apple device too.

**Comment by sheetalj2205** [comment]
> @sheetalj2205 add one screenshot for non-apple device too.

done

File: src/components/Carousel.jsx (hunk 1 above)
Comment (Anurag5086) [reply src/components/Carousel.jsx:16]: Okay! Looks like others commits have also come. Fine no problem.

**Review by Anurag5086 — APPROVED** [review]

**Comment by Anurag5086** [comment]
looks good! 👍

**Review by aryasoni98 — APPROVED** [review]

**Comment by sheetalj2205** [comment]
I am not getting scores for this PR. Even the changes that I have made is not visible.

**Pull request merged**